Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -72,6 +72,9 @@ public static object OnNewRepositorySerializerEvent(NewRepositorySerilizerEventA
         public  void SaveToFile(RepositoryItemBase ri, string FileName)
         {
             string txt = SerializeToString(ri);
+
+            string LongPathPrefix = @"\\?\";
```

Comment:
1- instead mentioning this LongPathPrefix in so many places, please add function so SolutionRepository called AdjustLongFilePath(string path) which will do the adjustment genericly and call it where needed

2- when declaring internal field it should start with lower letter like "longPathPrefix" and not "LongPathPrefix"